database/migration/src/main/resources/changelog.xml

```diff
@@ -90,6 +90,7 @@
     <include file="changesets/enlarge_docker_hash_store_entry_id.xml" relativeToChangelogFile="true" />
     <include file="changesets/enlarge_workflow_store_entry_id.xml" relativeToChangelogFile="true" />
     <include file="changesets/enlarge_sub_workflow_store_entry_id.xml" relativeToChangelogFile="true" />
+    <include file="changesets/change_table_owner_for_workflows_app_cromwell.xml" relativeToChangelogFile="true" />
```

Comment:
I think `for_workflow_app` is possibly unnecessary in the name, though I do like documenting _why_ we're doing it somewhere 🤔